code/game/objects/items/plushes.dm

```diff
@@ -97,6 +108,43 @@
 
 	return ..()
 
+/obj/item/toy/plush/proc/set_snowflake_from_config(id)
+	var/list/configlist = CONFIG_GET(keyed_list/snowflake_plushies)
+	var/list/jsonlist = configlist[id]
+	ASSERT(jsonlist)
+	jsonlist = json_decode(jsonlist)
+	if(jsonlist["inherit_from"])
+		var/path = text2path(jsonlist["inherit_from"])
+		if(!ispath(path, /obj/item/toy/plush))
+			stack_trace("Invalid path for inheritance")
+		else
+			var/obj/item/toy/plush/P = new path		//can't initial() lists
+			name = P.name
+			desc = P.desc
+			icon_state = P.icon_state
+			item_state = P.item_state
+			icon = P.icon
+			squeak_override = P.squeak_override
+			attack_verb = P.attack_verb
+			gender = P.gender
+	if(jsonlist["name"])
+		name = jsonlist["name"]
+	if(jsonlist["desc"])
+		desc = jsonlist["desc"]
+	if(jsonlist["gender"])
+		gender = jsonlist["gender"]
+	if(jsonlist["icon_state"])
+		icon_state = jsonlist["icon_state"]
+		item_state = jsonlist["item_state"]
+		icon = 'config/plushies/sprites.dmi'
+	if(jsonlist["attack_verb"])
+		attack_verb = jsonlist["attack_verb"]
+	if(jsonlist["squeak_override"])
+		squeak_override = jsonlist["squeak_override"]
```

Comment:
Forgot about this.
Make sure each squeak override key is converted from text to file.
Needless to restate my worries with config squeak overrides, but they should use sounds from a dedicated config/plushies/squeak_overrides folder (only add a readme inside for the repo version please).

Reply:
it worked without text2file conversion for some reason when i tested it

Reply:
Umh, nice.